packages/yourloops/components/patient-data/patient-data.hook.ts

```diff
@@ -221,6 +219,25 @@ export const usePatientData = (): usePatientDataResult => {
       setLoadingData(false)
     }
   }
+  const calculedBgDate = (dateBg: string[]): number => {
+    const dateRangeSet = new Set(dateBg)
+    if (dateRangeSet.size >= 14) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏14 could be set in a const such as `const FOURTEEN_DAYS = 14`